fslib/ui_filepicker.py

```diff
@@ -156,9 +188,19 @@ def retranslateUi(self, Dialog):
         Dialog.setWindowTitle(_translate("Dialog", "Dialog"))
         self.ui_other_fs.setText(_translate("Dialog", "OTHER"))
         self.ui_label_Lookin.setText(_translate("Dialog", "Look in:"))
-        self.ui_mkdir.setText(_translate("Dialog", "MD"))
+        self.ui_mkdir.setText(_translate("Dialog", "New Folder"))
+        self.ui_Action.setText(_translate("Dialog", "Open"))
+        self.ui_Cancel.setText(_translate("Dialog", "Cancel"))
         self.ui_FileList.setSortingEnabled(True)
         self.ui_label_filename.setText(_translate("Dialog", "File name: "))
         self.ui_label_filetype.setText(_translate("Dialog", "Files of type:"))
-        self.ui_Action.setText(_translate("Dialog", "Open"))
-        self.ui_Cancel.setText(_translate("Dialog", "Cancel"))
+
+
+if __name__ == "__main__":
```

Comment:
likly there is a pyuic5 -x option which adds this, but we don't use this.

please remove this section

converting by  pyuic5 should be enough

Reply:
Got it. Will make the changes